docs/proposals/0003-avoid-DOM-naming-props.md

```diff
@@ -0,0 +1,123 @@
+
+The structure of this rules is the following one:
+
+```
+...
+"react/forbid-component-props": [<enabled>, { "forbid": [<string>|<object>] }]
+...
+
+```
+
+This rule has two arguments:
+
+-   The `enabled` parameter determines the type of message that must be displayed once the rule is applied. Normally, it is set as `"error"` message.
+-   The `forbid` parameter is an string or object array where you have to specify the prop names that you want to avoid. Besides, you can also customize the array specifying the property name, an optional custom message, and a component whitelist:
+
+```
+{
+  "propName": "someProp",
+  "allowedFor": [SomeComponent, AnotherComponent],
+  "message": "Avoid using someProp"
+}
+
+```
+
+## Detailed design
+
+In order to set this `eslint` rule in the `.eslintrc.js` file, we have to follow two steps in order to determain which DOM attributes must be set up in the rule:
+
+1. In the first step, we have to determine which DOM attribute names must be set in the `eslint` rule because as I said before there is not any rule that check it automatically without specifying the DOM attribute names. In Clay components, there are some props like `title`, `value`, `length` that must be indicated in the rule. Moreover, you can also take a view of the [DOM attributes list](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes).
+
+2. Once we have determine the DOM attribute name list, the next step is building the rule. There are different ways of creating a rule: simple way and the complete one.
+
+### 1. Simple Way
+
+In the simple way, you can determine only the chosen DOM attribute names in a string array like the following example:
+
+```
+'react/forbid-component-props': [
+	'error', {
+		forbid: ['length', 'title', 'value'],
+	},
+],
+```
+
+Once you insert this example in `.eslintrc.js` file and save it, then go to the console and execute the following command:
+
+```
+yarn lint
+```
+
+It will show you all the files where Clay components have got prop names like DOM ones indicated in the string array.
+
+### 2. Complete Way
+
+In the complete way, you can determine only the chosen DOM attributes in an objects array like the following example:
+
+```
+'react/forbid-component-props': [
+	'error', {
+		forbid: [{
+        message: "In this file includes title prop",
+        propName: "title",
+    },
+    {
+        message: "In this file includes value prop",
+        propName: "value",
+        }],
+	},
+],
```

Comment:
just one typo, I think the formatting broke here.

Reply:
Hey @matuzalemsteles! I've just check it and I think that it is CLI GitHub visualisation problem. I'm going to change a white space in order to maintain the formatting consistency

Reply:
I have made some changes in formatting and the problem occurs GitHub CLI code review that the formatting broke